plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
```

Comment:
Yep thats perfectly fine. I think it is 2 different ways of achieving the same thing. Let's discuss tomorrow and decide on a solution.